gym/utils/env_checker.py

```diff
@@ -16,21 +17,28 @@
 import numpy as np
 
 import gym
-from gym import logger, spaces
+from gym import logger
+from gym.spaces import Box, Dict, Discrete, Space, Tuple
 
 
-def _is_numpy_array_space(space: spaces.Space) -> bool:
-    """
-    Returns False if provided space is not representable as a single numpy array
-    (e.g. Dict and Tuple spaces return False)
-    """
-    return not isinstance(space, (spaces.Dict, spaces.Tuple))
+def _is_numpy_array_space(space: Space) -> bool:
+    """Checks if a space can be represented as a single numpy array (e.g. Dict and Tuple spaces return False).
 
+    Args:
+        space: The space to check
 
-def _check_image_input(observation_space: spaces.Box, key: str = "") -> None:
+    Returns:
+        Returns False if the provided space is not representable as a single numpy array
     """
-    Check that the input adheres to general standards
-    when the observation is apparently an image.
+    return not isinstance(space, (Dict, Tuple))
+
+
+def _check_image_input(observation_space: Box, key: str = ""):
+    """Check that the input adheres to general standards when the observation is apparently an image.
```

Comment:
I had an issue with the "apparently". 
IIRC, `env_checker` checks whether the observation space *looks* like an image (i.e. the shape is something like AxBx3), and when it *apparently* is, it checks the standards. In contrast to that, this function *does not* check whether the space appears to represent images, it always checks the stadards.

```suggestion
    """Check whether an observation space of type :class:`Box` adheres to general standards for spaces that represent images.
    
    It will check that:
    - The datatype is ``np.uint8``
    - The lower bound is 0 across all dimensions
    - The upper bound is 255 across all dimensions
```